view-user/src/components/common/PrizeCardList/PrizeCardList.module.css

```diff
@@ -0,0 +1,5 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(3, 30vw);
+  justify-content: center;
+}
```

Comment:
PrizeCardが中央からずれていたのと、行間の余白が無いことが気になりました。
| 修正前 | 修正後 |
|--|--|
|<img width="300" alt="image" src="https://github.com/user-attachments/assets/8ebca5a6-cac7-4543-8edd-0025a80ef70a">|<img width="300" alt="image" src="https://github.com/user-attachments/assets/d448a44f-33c4-4f45-8986-54f2f5c5d61b">|


```suggestion
  place-items: center;
  align-items: start;
  row-gap: 5vw;
}
```

Reply:
ありがとうございます！
place-itemsの存在を初めて知りました！！
https://github.com/NUTFes/nutfes-Bingo/pull/235/commits/13205a8e0a210cd5ef7a38f4c43d7e599a60bd8b